pkg/operator/api/authz.go

```diff
@@ -24,12 +22,7 @@ import (
 
 // authzRequest ensures that the requesting identity resides in the same
 // namespace as that of the requested namespace.
-func (a *apiServer) authzRequest(ctx context.Context, namespace string) error {
-	spiffeID, ok, err := spiffe.FromGRPCContext(ctx)
-	if err != nil || !ok {
-		return status.New(codes.PermissionDenied, "failed to determine identity").Err()
-	}
```

Comment:
do not move this to utils. have a function here itself to get spiffe ID or use the provided `FromGRPCContext` function from the module directly in `api.go`.

Reply:
Though I believe that GetSpiffeIdFromContext can be used at other places as well and better placed in utils. But, I have moved that to this file itself.

Reply:
I agree utils may not be the best package, but I do like the idea of it being in _a_ shared package. What about some security package?

Reply:
A security package if needed works.